client/doc.go

```diff
@@ -0,0 +1,52 @@
+/*
+Package client provides transport-agnostic logic to retreive and verify
+randomness from drand, including retry, validation, caching and
+optimization features.
+
+Example:
+
+	import (
+		"context"
+		"encoding/hex"
+		"fmt"
+
+		"github.com/drand/drand/client"
+	)
+
+	var chainHash, _ = hex.DecodeString("8990e7a9aaed2ffed73dbd7092123d6f289930540d7651336225dc172e51b2ce")
+
+	func main() {
+		c, err := client.New(
+			client.From(...), // see concrete client implementations
+			client.WithChainHash(chainHash),
+			)
+
+		// e.g. use the client to get the latest randomness round:
+		r := c.Get(context.Background(), 0)
+
+		fmt.Println(r.Round(), r.Randomness())
+	}
+
+
+The "From" option allows you to specify clients that work over particular
+transports. HTTP, gRPC and libp2p PubSub clients are provided. Note that you
```

Comment:
🙏 can we link to the doc pages for these clients?

Reply:
linked